### ISSUE TITLE
Not an issue but request

### ISSUE THREAD

**Issue by maciej-szalaj** [benct/lovelace-xiaomi-vacuum-card#50]
I really like your card - it is fantastic. Any chance to add second row of 6 icons to use it for specific room cleaning command?

**Comment by benct** [comment]
Yes, I would like to add the possibility to add custom buttons. Follow up in #26.
